src/Microsoft.DocAsCode.MarkdigEngine/Validation/ValidationExtension.cs

```diff
@@ -25,6 +28,13 @@ public void Setup(MarkdownPipelineBuilder pipeline)
 
             pipeline.DocumentProcessed += document =>
             {
+                if (InclusionContext.IsInclude 
```

Comment:
Could validation library handle this? Other component does not need this, and validation library has enough knowledge to calculate this.

Reply:
The purpose I add "IsInclude" condition here is because the Yaml file could contain 'inline' (not included) markdown property, and in such case, we actually don't have to tag 'yamlMime' as we would skip validation when markdown inline.

Reply:
This looks very much like the logic that should be handled on validation library side - SDP should be general purposed that the insertion of YamlMime should not depend on whether the Markdig piece is inline or not.

Reply:
Renze and I talk over this offline, the reason I add the logic in docfx side is because validation service could not retrieve root file yaml mime with the file path provided. It is a relative path, validation library could not retrieve yaml mime directly due to lack of docfx file locale in the context.